ObservatoryCore/PluginManagement/PluginManager.cs

```diff
@@ -54,21 +54,55 @@ private PluginManager()
 
             var core = new PluginCore();
 
+            List<string> loadErrors = new();
+            Dictionary<IObservatoryWorker, PluginException> workerErrors = new();
+
             foreach (var plugin in workerPlugins.Select(p => p.plugin))
             {
-                LoadSettings(plugin);
-                plugin.Load(core);
+                try
+                {
+                    LoadSettings(plugin);
+                    plugin.Load(core);
+                }
+                catch (PluginException ex)
+                {
+                    workerErrors.Add(plugin, ex);
+                }
+            }
+
+            foreach(var error in workerErrors)
+            {
+                loadErrors.Add($"{error.Value.PluginName}: {error.Value.UserMessage}");
```

Comment:
Why not just add the error directly to loadErrors in catch() (also for notifiers)? Then you can reduce the Dictionary to just a set, which would also simplify removal linq statement.

Also, consider a method for generating the error string as well, since they're identically formatted. 

In the end, reduces duplicate code.

Reply:
Good call, will do.

Reply:
_Way_ better: 692b545